MessageBird/MessageBird.csproj

```diff
@@ -1,29 +1,18 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net40;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net6.0</TargetFrameworks>
     <Product>MessageBird</Product>
     <Title>MessageBird</Title>
     <Company>MessageBird</Company>
-    <Copyright>Copyright © 2019</Copyright>
-    <FileVersion>3.1.0.0</FileVersion>
-    <InformationalVersion>3.1.0.0</InformationalVersion>
-    <Version>3.1.0.0</Version>
-    <AssemblyVersion>3.1.0.0</AssemblyVersion>
-    <LangVersion>4</LangVersion>
+    <Copyright>Copyright © 2022</Copyright>
+ 
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
   </PropertyGroup>
-
-  <!-- Conditionally obtain references for the .NET Framework 4.0 target -->
-  <ItemGroup Condition=" '$(TargetFramework)' == 'net40' ">
-  </ItemGroup>
-
   <ItemGroup>
     <PackageReference Include="JWT" Version="8.2.3" />
```

Comment:
Really like the switch to target .net6.0.
Be ware that targetting .net6.0 might leave customers running on .NET Framework behind, as you are no longer targetting .netstandard 6.0.
https://docs.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-1-0#net-standard-not-deprecated

Is it possible to remove this dependency to JWT and switch over to System.Text.Json? That would result in a library with only framework dependencies.
JWT package still depends on Newtonsoft.Json, there is an issue on their side to replace it: https://github.com/jwt-dotnet/jwt/issues/364

Reply:
Yes. I think it's time we leave the .NET Framework behind. 
Well make a major change release and then people have. Acho ice to stay on old versions. 

About your idea of using System's Jason.. it looks like a very good one. 
We have limited c# knowledge these days over here so if you can do a PR or point us on the right direction it would be amazing